frontend/src/routes.tsx

```diff
@@ -98,6 +101,12 @@ export const AppRoutes = (): JSX.Element => {
 	return (
 		<BrowserRouter>
 			<Routes>
+				<Route
```

Comment:
did we settle on using a separate route of /public?

Reply:
Yes, this one does not have the Depends(get_current_user) for methods but will always only return datasets that are public, and does not have routes that edit or create datasets. 

I did think that it might have been a good idea to have made a dependency that checks public, but it didn't seem absolutely necesary.

Reply:
I just pushed a change that I thought would work for this, but so far it still redirects to keycloak. Working on this.

Reply:
I think this is fixed now.